scripts/india_udise/udise_schools_having_computer_available/preprocess.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Classes and methods to import Number of Schools having Computer Available from Unified District Information System for Education (UDISE)"""
+
+__author__ = ["Thejesh GN (i@thejeshgn.com)"]
+
+import os
+import sys
+import json
+import csv
+import pandas as pd
+import numpy as np
+import requests
+import time
+from os import path
+from india_udise.common.base_school_data import UDISEIndiaSchoolDataLoaderBase
+from india_udise.common.base_school_data import BASE_DATA_FOLDER
+
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+
+
+class UDISESchoolsHavingComputerAvailable(UDISEIndiaSchoolDataLoaderBase):
+
+    def _get_base_name(self, data_row):
+        name = "Count_School_HasComputerAvailable"
```

Comment:
Can we drop the 'Available' suffix to keep with the practice of using the property value 'Computer' in the name?

Reply:
Done